embrio-async/macros/src/lib.rs

```diff
@@ -12,6 +12,25 @@ use syn::{
     ReturnType, TypeImplTrait, TypeParam, TypeParamBound, TypeReference,
 };
 
+// A `.await` expression is transformed into,
+//
+// {
+//     let mut pinned = { ... }
```

Comment:
Maybe replace the `...` here and below with `/* the input expression */` and `/* the input block */`